src/api.js

```diff
@@ -22,4 +22,15 @@ function fetchCommentsByArticleId(article_id) {
   });
 }
 
-export { fetchArticles, fetchArticleById, fetchCommentsByArticleId };
+function updateArticleLikes(article_id) {
+  return api
+    .patch(`/articles/${article_id}`, {
+      inc_votes: 1,
```

Comment:
why have you hard coded this ? Will this not always increment likes even when you want to decrement them ?